src/Components/DrumPad/index.js

```diff
@@ -34,7 +34,7 @@ const DrumPad = ({ count, layout, swapBeat }) => {
               <td
                 key={beatNum}
                 onClick={() => {
-                  console.log({ rowNum, beatNum });
+                  console.log({ rowNum, beatNum, hasBeat });
```

Comment:
- [ ] Can you remove this console.log entirely? I accidentally left it in after I was debugging something :)